app/scripts/modules/amazon/src/serverGroup/configure/serverGroupConfiguration.service.ts

```diff
@@ -474,10 +474,10 @@ export class AwsServerGroupConfigurationService {
     const vpcLoadBalancers = this.getVpcLoadBalancerNames(command);
     const allTargetGroups = this.getTargetGroupNames(command);
 
-    if (currentLoadBalancers && command.loadBalancers && !currentLoadBalancers.includes('${')) {
```

Comment:
`!currentLoadBalancers.includes('${')` was always true because `currentLoadBalancers`, an array, almost never includes `'${'` as an item. Probably should've been something like `!currentLoadBalancers.some(lb => lb.includes('${'))` but I don't think we want that now either.